File: ERP/src/main/java/edu/academia/ERP/dto/StudentResponse.java
package edu.academia.ERP.dto;

public record StudentResponse(
        Long student_id,
        String roll_number,
        String first_name,
        String last_name,
        String photograph_path,
        Double cgpa,
        int total_credits,
        String graduation_year,
        Long domain,
        Long specialisation,
        Long placement_id
) {
}

File: erp-ft/src/models/student/StudentResponse.js
class StudentResponse {
    constructor(data) {
      this.student_id = data.student_id
      this.roll_number = data.roll_number;
      this.first_name = data.first_name;
      this.last_name = data.last_name;
      this.photograph_path = data.photograph_path;
      this.cgpa = data.cgpa
      this.total_credits = data.total_credits;
      this.graduation_year = data.graduation_year;
      this.domain = data.domain;
      this.specialisation = data.specialisation;
      this.placement_id = data.placement_id;
    }

    
  }
  
  export default StudentResponse;
  